skops/card/_model_card.py

```diff
@@ -366,6 +368,33 @@ def add_metrics(self, **kwargs: str) -> "Card":
             self._eval_results[metric] = value
         return self
 
+    def add_feature_importances(self, feature_importances) -> "Card":
+        """Visualize permutation importance.
+
+        Parameters
+        ----------
+        self : object
+            Card object.
+        feature_importances : sklearn.utils.Bunch
+            Output of sklearn.inspection.permutation_importance()
+
+        Returns
+        -------
+        self : object
+            Card object.
+        """
+
+        importances = pd.DataFrame(
+            feature_importances.importances.T,
+            columns=self.model.feature_names_in_,
+        )
+        ax = importances.plot.box(vert=False, whis=10)
+        ax.set_title("Permutation Importances")
+        ax.set_xlabel("Decrease in accuracy score")
+        plt.savefig("feature_importances.png")
+        self.add_plot(**{"Feature Importances": "feature_importances.png"})
+        return self
```

Comment:
here pandas is only used to handle plotting, I think we can avoid this and create the plot ourself and remove the pandas import here.